config/config.go

```diff
@@ -989,6 +998,24 @@ func SetupViper(v *viper.Viper, filename string) {
 	v.SetEnvPrefix("PBS")
 	v.AutomaticEnv()
 	v.ReadInConfig()
+
+	// Migrate config settings to maintain compatibility with old configs
+	migrateConfig(v)
+}
+
+func migrateConfig(v *viper.Viper) {
+	//v.RegisterAlias("stored_requests.directorypath", "stored_requests.filesystem.directorypath")
```

Comment:
Why is this commented out?

Reply:
I'll remove it. This would have been the natural method to migrate config settings so I tried it first, but it does not work due to `stored_requests.filesystem` having been promoted from a bool to a map.